### PR TITLE
Updated debian packaging

### PR DESCRIPTION
**Pull request by aviau**
updated Debian packaging, now with the packaged javascript libraries.

```make dpkg``` does the magic ;)

### CODE REVIEW

File: packages/debian/control [hunk 1]
```diff
@@ -26,14 +49,15 @@ Homepage: https://www.mailpile.is/
...
+Depends: ${misc:Depends},
+         mailpile-server
 Description: email client with user-friendly encryption and privacy features
  Mailpile is a modern, fast web-mail client with user-friendly encryption and
  privacy features. Mailpile places great emphasis on providing a clean, elegant
  user interface and pleasant user experience. In particular, Mailpile aims to
  make it easy and convenient to receive and send PGP encrypted or signed e-mail.
  .
- This package installs the desktop client.
+ This is package provides the currenty recommended install for Mailpile.
```
Comment (BjarniRunar) [inline packages/debian/control]: This is typo. :)

File: packages/debian/copyright [hunk 2]
```diff
@@ -1,10 +1,58 @@
...
+                bower_components/purl
+                bower_components/qtip2
+                bower_components/rebar
+                bower_components/select2
+                bower_components/typeahead.js
+                bower_components/underscore
+
 
 Files: *
 Copyright: 2011-2015 Bjarni R. Einarsson, Mailpile ehf and friends.
```
Comment (BjarniRunar) [inline packages/debian/copyright]: 2016 ... there are more places where this is wrong.

**Comment by BjarniRunar** [comment]
Merged, see minor nits above for including in next iteration (assuming there is one).

File: packages/debian/control (hunk 1 above)
Comment (aviau) [inline packages/debian/control]: Haha :)

**Pull request merged**